includes/extra_configures/set_time_zone.php

```diff
@@ -8,21 +8,19 @@
 /*
  * Set time zone
 */
-  // put your timezone here. Refer to http://www.php.net/manual/en/timezones.php
-  $TZ = '';  // eg: 'Europe/Oslo'
+// put your timezone here. Refer to http://www.php.net/manual/en/timezones.php
+$TZ = '';  // eg: 'Europe/Oslo'
 
-
-
-  /**
-   * MAKE NO CHANGES BELOW THIS LINE
-   *
-   * The following will take the timezone you specified above and apply it in your store.
-   * If you didn't specify one, it will try to use the setting from your server's PHP configuration
-   */
-  if ($TZ == '') {
+/**
+* MAKE NO CHANGES BELOW THIS LINE
+*
+* The following will take the timezone you specified above and apply it in your store.
+* If you didn't specify one, it will try to use the setting from your server's PHP configuration
+*/
+if ($TZ === '') {
     $TZ = date_default_timezone_get();
-  }
-  if ($TZ != '') {
+}
+if ($TZ !== '') {
     putenv('TZ=' . $TZ);
-    @date_default_timezone_set($TZ);
-  }
+    date_default_timezone_set($TZ);
```

Comment:
For reference, the `@` suppressor was initially used here due to inconsistencies in PHP 5's date functions. Some versions would fail based on compiler settings or even sub version releases.

Should be fine without it nowadays.